sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveMetastoreCatalog.scala

```diff
@@ -585,34 +589,6 @@ private[hive] class HiveMetastoreCatalog(hive: HiveContext) extends Catalog with
         }
 
       case p: LogicalPlan if p.resolved => p
-
-      case p @ CreateTableAsSelect(db, tableName, child, allowExisting, None) =>
```

Comment:
@chenghao-intel Why `hasStorageSpec` was not implemented right? Also `CREATE TABLE USING` will use `CreateTableUsingAsSelect`. Also, a little bit off topic, can you remind me why the `desc` in `CreateTableAsSelect` is an `Option`?